db/models.py

```diff
@@ -50,3 +53,75 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession, on_delete=models.CASCADE, related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order, on_delete=models.CASCADE, related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["movie_session", "row", "seat"], name="unique_ticket"
+            )
+        ]
+
+    def __str__(self) -> str:
+        movie = self.movie_session.movie.title
+        movie_session = self.movie_session.show_time
+        return (f"{movie} {movie_session}"
+                f" (row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        super().clean()
```

Comment:
you actually don't need to call super() here because models.Model method clean is just a placeholder to be overriden in child methods